beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1812,7 +1812,7 @@ pub static KZG_VERIFICATION_BATCH_TIMES: LazyLock<Result<Histogram>> = LazyLock:
 pub static KZG_VERIFICATION_DATA_COLUMN_SINGLE_TIMES: LazyLock<Result<Histogram>> =
     LazyLock::new(|| {
         try_create_histogram_with_buckets(
-            "kzg_verification_data_column_single_seconds",
+            "beacon_kzg_verification_data_column_single_seconds",
             "Runtime of single data column kzg verification",
```

Comment:
Note that we also have similar metrics (e.g. `kzg_verification_single_seconds`) for Deneb blobs, but we can probably leave them as it is, as they will eventually be replaced by the PeerDAS ones.

Reply:
Found 3 `kzg_data_column` metrics below. What about `data_availability_cache_size` metrics above?